samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -124,7 +124,7 @@ func WithLogger(logger logr.Logger) Option {
 }
 
 // samplingStrategyFetcher creates a Option that initializes sampling strategy fetcher.
```

Comment:
1. The comment should start with `WithSamplingStrategyFetcher` instead of `samplingStrategyFetcher`.
2. Can you add information when this option should be used?